packages/jest-core/src/runGlobalHook.ts

```diff
@@ -45,17 +46,26 @@ export default async ({
 
       const transformer = new ScriptTransformer(projectConfig);
 
-      await transformer.requireAndTranspileModule(modulePath, async m => {
-        const globalModule = interopRequireDefault(m).default;
+      try {
```

Comment:
ignore whitespace

Reply:
There is a link to whitespace-free diff: https://github.com/facebook/jest/pull/11003/files?w=1